### DISCUSSION TITLE
how can I read `rectangular array` entity use ezdxf api?

### DISCUSSION THREAD

**Discussion by xinkunZ** [mozman/ezdxf#1153] [Q&A]
I have a dxf like this,  it has a rectangular array with some row ang column,  how can I read this item?
I have try to use INSETS entity, but it's properties seems does not match the graphics 

![image](https://github.com/user-attachments/assets/101154e1-ac67-4409-bbdd-86b07ffa73a1)

**Comment by mozman** [comment]
These arrays are not special DXF entities, these are simple block references. Use the `ezdxf browse` command to investigate the DXF structure.
